### ISSUE TITLE
Seemingly impossible to change the foreground text color

### ISSUE THREAD

**Issue by ocecaco** [ubergeek42/weechat-android#414]
I tried to change my color scheme to a lighter color scheme such as Lucius White or Solarized Light, but changing the color scheme does not seem to affect the color of the text in the chat. I have also tried creating a theme file where I set all colors to bright red (0xFF0000), but this also does not affect the text that has been written by people in the channel.

This is the theme file that I tried to use:
```
NAME = My Theme

DEFAULT    = 0xFF0000
DEFAULT_BG = 0x0f1315

color0   = 0xFF0000
color1   = 0xFF0000
color2   = 0xFF0000
color3   = 0xFF0000
color4   = 0xFF0000
color5   = 0xFF0000
color6   = 0xFF0000
color7   = 0xFF0000
color8   = 0xFF0000
color9   = 0xFF0000
color10  = 0xFF0000
color11  = 0xFF0000
color12  = 0xFF0000
color13  = 0xFF0000
color14  = 0xFF0000
color15  = 0xFF0000

separator = 0xFF0000
chat = 0xFF0000
chat_time = 0xFF0000
chat_time_delimiters = 0xFF0000
chat_prefix_error = 0xFF0000
chat_prefix_network = 0xFF0000
chat_prefix_action = 0xFF0000
chat_prefix_join = 0xFF0000
chat_prefix_quit = 0xFF0000
chat_prefix_more = 0xFF0000
chat_prefix_suffix = 0xFF0000
chat_buffer = 0xFF0000
chat_server = 0xFF0000
chat_channel = 0xFF0000
chat_nick = 0xFF0000
chat_nick_self = 0xFF0000
chat_nick_other = 0xFF0000
chat_host = 0xFF0000
chat_delimiters = 0xFF0000
chat_highlight = 0xFF0000
chat_read_marker = 0xFF0000
chat_text_found = 0xFF0000
chat_value = 0xFF0000
chat_prefix_buffer = 0xFF0000
chat_tags = 0xFF0000
chat_inactive_window = 0xFF0000
chat_inactive_buffer = 0xFF0000
chat_prefix_buffer_inactive_buffer = 0xFF0000
chat_nick_offline = 0xFF0000
chat_nick_offline_highlight = 0xFF0000
chat_nick_prefix = 0xFF0000
chat_nick_suffix = 0xFF0000
emphasis = 0xFF0000
chat_day_change = 0xFF0000
```

**Comment by ocecaco** [comment]
Perhaps this will be fixed by #408 ?

**Comment by oakkitten** [comment]
we are closing some of the issues that weren't active in a while or may be irrelevant for the latest version (1.0). if you think that this issue is still applicable, please leave a comment.
